src/OutputWriters/OutputWriters.jl

```diff
@@ -0,0 +1,18 @@
+module OutputWriters
+
+export
+    write_output, read_output,
+    has_reference, has_array_refs,
+    JLD2OutputWriter, FieldOutput, FieldOutputs,
+    NetCDFOutputWriter, write_grid,
+    Checkpointer, restore_from_checkpoint
+
+using Oceananigans
+using Oceananigans: AbstractOutputWriter
```

Comment:
Should `AbstractOutputWriter` be introduced in this submodule, rather than in `Oceananigans`? I think concept encapsulation is good.

Reply:
See above comment.